docs/guides/runtime-options-overview.mdx

```diff
@@ -199,14 +507,12 @@ For more information about the primitive options, refer to the
 </Tabs>
 
 
+### Error suppression
```

Comment:
Since we already have entire pages on error suppression and mitigation, I don't think we need to repeat them here. We can instead add `max_execution_time` as another commonly used option.